src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1279,13 +1449,14 @@ const draftBlogPosts = await getCollection('blog', ({ data }) => {
 
 ### `getEntry()`
 
-<p><Since v="2.5.0" /></p>
+<p>
 
 **Types:**
-
 - `(collection: string, contentSlugOrDataId: string) => CollectionEntry<collection>`
 - `({ collection: string, id: string }) => CollectionEntry<collection>`
 - `({ collection: string, slug: string }) => CollectionEntry<collection>`
+<Since v="2.5.0" />
+</p>
```

Comment:
I'm not sure about this one. Since we have a list of types the rendering might seem odd. I tried to place `<Since />` before `Types:` but we have a new line between the two. I think the Markdown parser ignores the `<p>` tag around because of the list inside. So I'm not sure what is the best option here.

Reply:
Oh boy, yes, had no idea we’d allowed someone to have function overloads like this 😅 

I _think_ it might look ok if `<Since>` appears first in this case?

<img width="753" alt="getEntry docs with added in version displayed above the list of valid types" src="https://github.com/user-attachments/assets/dfc0ef73-181c-4148-b9d5-b6e7c57ee5c3">

Reply:
I felt like the spacing was bigger when I tried... I must have tested too quickly. So yes, it seems to be the best option when using multiple types!

Reply:
It seems I'm struggling with this one. In all my tests, `Types:` is always rendered in its own `<p>` tag. So the spacing does not match your screenshot.

For example, the following should render both in a single `<p>` tag, but no... `Types:` is stubborn 😬 and has is own `<p>` tag in the end.

```mdx
<p>

<Since v="2.5.0" />
**Types:**
</p>
```

@delucis It was a live edit for your screenshot? The best I got is:

![Illustration of the too important spacing between since and types](https://github.com/user-attachments/assets/6fc2574a-21b9-4032-a4a3-80c6840442bd)

Is there any other trick I don't know about Markdown rendering? 😅 (Thanks by the way for the new line explanation! Erika had already told me about it for the MD images inside `figure`... but I'm not used to this syntax so I had forgotten... 👀 )